index.html

```diff
@@ -325,5 +325,21 @@ <h2 class="kn_1">
     </li>
   </ul>
 </section>
+    <footer id="contact" class="contact">
+      <div class="container-contact">
+        <h2 class="first-text-contac"> Contact me</h2>
+        <p class="second-text-contact">If you have an application you are interested in developing, a feature that you need built or a
+            project that
+            needs coding. I’d love to help with it </p>
+      </div>
+          
+      <form class="container-form" action="https://formspree.io/f/xgedzbjq" method="post" novalidate>
```

Comment:
Kindly remove the ` novalidate ` attribute as now all validation you add will be ignored.